sdk/attestation/ci.yml

```diff
@@ -30,7 +30,7 @@ stages:
       CtestRegex: azure-security-attestation.*
       LiveTestCtestRegex: azure-security-attestation.*
       LineCoverageTarget: 90
-      BranchCoverageTarget: 45
+      BranchCoverageTarget: 42
```

Comment:
=( .  It happens.

Reply:
Most of this is because the new APIs are all live-only. I guess I could add tests that don't actually check the results of the operation, but that seems weird.